CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 
 - Added support for `--diff-format` option under `kart show` command to display different level of detail depending on the user's needs.  [#721](https://github.com/koordinates/kart/issues/721)
 
+- Resolved issue [#805](https://github.com/koordinates/kart/issues/805) by ensuring the `add-dataset` command creates tracking triggers for committed tables, enabling future changes to be detected and committed.
```

Comment:
add-dataset is a new feature in this release (L10) so we don't need to mention this - the whole feature is new